dags/common/utils.py

```diff
@@ -149,3 +150,16 @@ def preserve_cdata(article: str):
 
 def parse_to_ET_element(article: str):
     return ET.fromstring(preserve_cdata(article))
+
+
+def parse_without_names_spaces(xml: str):
+    it = ET.iterparse(StringIO(xml))
+    for _, el in it:
+        _, _, el.tag = el.tag.rpartition("}")
+    root = it.root
+    return root
+
+
+def get_text_value(element: ET.Element):
+    if element is not None:
```

Comment:
`if element:` won't work?

Reply:
No, because the bool(element) will return True

Reply:
<img width="394" alt="review" src="https://user-images.githubusercontent.com/36658177/225297721-edb62eb7-bbdc-4855-b570-88dd323389b7.png">

Reply:
it's ok, sometimes it's like that ;)